apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -981,8 +981,9 @@ const Ticket = new GQLListSchema('Ticket', {
             },
             {
                 type: 'BTreeIndex',
-                fields: ['assignee', 'executor', 'organization'],
-                name: 'ticket_assignee_exec_org',
+                fields: [ 'organization', 'assignee', 'executor'],
+                name: 'ticket_org_assignee_exec',
+                condition: 'Q(deletedAt__isnull=True)',
```

Comment:
Seem's okay, but are you tying the index to a specific query from the application, right?
If I'm right, maybe it would be better to leave current index and add another one.
First of all you might check how popular this query is)

Reply:
Although I know the answer in advance) Because this query is used on the control room page)

Reply:
mobile app also uses same request :)

Reply:
modified it slightly: removed filter, but added "deletedAt" to the fileds
the query still uses this indedx, the cost of query is the same, but it could be beneficial not to limit this index by using the filter